src/web/app/pages-student/student-profile-page/student-profile-page.component.spec.ts

```diff
@@ -37,4 +39,67 @@ describe('StudentProfilePageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with a student field without information', () => {
+    const studentDetails: any = {
+      studentProfile: {
+        shortName: '',
+        email: '',
+        institute: '',
+        nationality: '',
+        gender: Gender,
+        moreInfo: '',
+        pictureKey: '',
+      },
+      name: '',
+      requestId: '',
+    };
+    component.student = studentDetails;
+    component.editForm = new FormGroup({
+      studentshortname: new FormControl(''),
+      studentprofileemail: new FormControl(''),
+      studentprofileinstitute: new FormControl(''),
+      studentnationality: new FormControl(''),
+      existingNationality: new FormControl(''),
+      studentgender: new FormControl(''),
+      studentprofilemoreinfo: new FormControl(''),
+    });
+    fixture.detectChanges();
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with values and a profile photo', () => {
+    const studentDetails: any = {
+      studentProfile: {
+        shortName: 'Ash',
+        email: 'ayush@nus.com',
+        institute: 'NUS',
+        nationality: 'Indian',
+        gender: Gender.MALE,
+        moreInfo: '',
```

Comment:
You should put in some values here, just to check if it's rendered properly. The empty `moreInfo` case is already covered in the previous test case